templates/docker-virt-manager.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Docker-Virt-Manager</Name>
+  <Repository>mber5/virt-manager:latest</Repository>
+  <Registry>https://hub.docker.com/r/mber5/virt-manager</Registry>
+  <Network>bridge</Network>
+  <Shell>bash</Shell>
+  <Privileged>true</Privileged>
+  <Support>https://forums.unraid.net/topic/77603-virt-manager-intel-gpu-tools-and-more-dockers/</Support>
+  <Project>https://virt-manager.org/</Project>
+  <Overview>GTK Broadway web UI for libvirt. The virt-manager application is a desktop user interface for managing virtual machines through libvirt. It primarily targets KVM VMs, but also manages Xen and LXC (linux containers). It presents a summary view of running domains, their live performance &amp;amp; resource utilization statistics. Wizards enable the creation of new domains, and configuration &amp;amp; adjustment of a domain&#x2019;s resource allocation &amp;amp; virtual hardware. Uses GTK3 Broadway (HTML5) backend--no vnc, xrdp, etc needed! Also has DARK MODE</Overview>
+  <Category>Tools:</Category>
+  <Beta>true</Beta>
+  <WebUI>http://[IP]:[PORT:8185]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/UnRaid/template/docker-virt-manager.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/virt-manager-icon.png</Icon>
+  <DonateText>If you like my work please consider Donating.</DonateText>
+  <DonateLink>https://interbiznw.github.io/</DonateLink>
```

Comment:
What work?

Reply:
Creating the template and getting a new functional and better UI/UX Virt manager in UNRAID again.

Reply:
I see nothing pointing to the fact that it's your image.

Reply:
Not sure what difference that makes, I took the time to create the template that apparently no one has created for this image...but apparently that is meaningless rite? Cool. Removed.

Reply:
Next time Ill just copy the image and put it in my own dockerhub 😂